modules/ROOT/pages/netsuite/netsuite-openair-connector.adoc

```diff
@@ -97,17 +86,30 @@ image::netsuite/netsuite-openair-connect-conf.png[connector config]
 
 You can use this connector in Anypoint Studio by adding it as a dependency in your Mule application.
 
+== Add the Connector to Your Project
 
-=== Install Connector in Studio
+Anypoint Studio provides two ways to add the connector to your Studio project: from the 
+Exchange button in the Studio taskbar or from the Mule Palette.
```

Comment:
Mule Palette view

Please make corrections from previous edits before submitting new text for editing.